### PR TITLE
Adding migrate to AAD API Spec.

### PR DESCRIPTION
**Pull request by ambhat**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json [hunk 1]
```diff
@@ -230,6 +230,54 @@
...
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "x-ms-long-running-operation": true,
```
Comment (anuchandy) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: Just curious, i didn't see 204 begin declared as a response type for any of the other POST LRO in the same swagger? If not intended, please remove it.

**Review by anuchandy — CHANGES_REQUESTED** [review]

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json (hunk 1 above)
Comment (anuchandy) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: @ambhat please confirm

**Comment by fearthecowboy** [comment]
ATTENTION REQUIRED.
-------------

We have changed the `default` branch in the `azure-rest-api-specs` repository to use the branch labeled `current`.

This PR is opened against the `master` branch.

You have two options: 
1. Keep this PR open, and complete the review process as normal, and once this PR is merged against the `master` branch, we (@olydis & @fearthecowboy)  will create a PR against the current branch and make any updates required to work in the newly reorganized default branch.  

2. Close this PR and create a new one. That would mean pulling in the changes from the new default branch `current` moving any files and switching to the configuration-based layout.  (You are more than welcome to choose option 2, but I don't expect anyone to do that)

Either way, the work you are currently doing against the previous default branch `master` will be the last PR against that branch that we will accept and all new PRs will go against the new default branch  `current`.

If you have any questions or concerns, ping @fearthecowboy or @olydis (or go ask on the `#swagger` channel in slack) ASAP and we can help you thru the process.

**Comment by salameer** [comment]
@ambhat this PR has been inactive for a while, please respond ASAP to  avoid having this PR closed due to inactivity

**Comment by ambhat** [comment]
Hi,
Sorry for the late reply. I was OOF for few days which caused the delay in response.

**Comment by ambhat** [comment]
Also, if there are no more comments apart from the above one (204), I want to get this PR merged rather than create a new branch etc.

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json (hunk 1 above)
Comment (anuchandy) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: thanks LGTM

**Review by anuchandy — APPROVED** [review]

**Pull request merged**